src/app/pages/checkout-payment/payment-payone-creditcard/payment-payone-creditcard.component.html

```diff
@@ -0,0 +1,65 @@
+<div [formGroup]="payoneCreditCardForm">
+  <p class="text-danger col-sm-offset-4" id="error"></p>
+  <!-- card number -->
+  <div class="row form-group">
+    <label class="col-form-label col-md-4" for="cardpanInput"
+      >{{ 'checkout.credit_card.number.label' | translate }}<span class="required">*</span></label
+    >
+    <div class="col-sm-6">
+      <span id="cardpan" class="inputIframe"></span>
```

Comment:
The styling of the iframes doesn't correspond to our standard styling, e.g. if the iframe has focus there is a black border instead of the blue one.

Reply:
The styling is provided by properties based configuration in the connector. It has to be updated there.